src/Nethermind/Nethermind.JsonRpc/JsonRpcUrl.cs

```diff
@@ -32,7 +32,7 @@ public JsonRpcUrl(string scheme, string host, int port, RpcEndpoint rpcEndpoint,
             Host = host;
             Port = port;
             RpcEndpoint = rpcEndpoint;
-            EnabledModules = enabledModules;
+            EnabledModules = enabledModules.Select(name => name.ToLower()).ToArray();
```

Comment:
could you explain why we're changing it?

Reply:
In some plases we check that "engine"(in lower case) module in this list. I thought it might cause similar problems in the future, so a better solution is to change it here

Reply:
Wouldn't it be better then to do: `EnabledModules = new HashSet<string>(enabledModules, StringComparer.InvariantCultureIgnoreCase)`